packages/react-query/src/useQuery.ts

```diff
@@ -7,6 +7,28 @@ import type {
 } from './types'
 import { useBaseQuery } from './useBaseQuery'
 
+export function queryOptions<
+  TQueryFnData = unknown,
```

Comment:
Since this is `query` specific, we would need similar helper for `infinite`, `mutation` and `queries`?

Reply:
probably not necessary for mutations, but for infinite: yes

Reply:
We don't even have the initialData overload for infiniteQuery 😅

Reply:
I would add a separate helper in a follow-up. This is not breaking so we can add it at any time I think